examples/01-Detailed-Examples/05-mesh-exploration.py

```diff
@@ -0,0 +1,197 @@
+"""
+.. _ref_mesh_exploration_example:
+
+Explore the mesh
+================
+In this script a static simulation is used as an example to show how to
+query mesh information such as connectivity, element IDs, element types and so on.
+"""
+
+###############################################################################
+# Perform required imports
+# ------------------------
+# Perform required imports. # This example uses a supplied file that you can
+# get by importing the DPF ``examples`` package.
+
+from ansys.dpf import post
+from ansys.dpf.post import examples
+from ansys.dpf.post.common import elemental_properties
+
+###############################################################################
+# Get ``Simulation`` object
+# -------------------------
+# Get the ``Simulation`` object that allows access to the result. The ``Simulation``
+# object must be instantiated with the path for the result file. For example,
+# ``"C:/Users/user/my_result.rst"`` on Windows or ``"/home/user/my_result.rst"``
+# on Linux.
+
+example_path = examples.download_all_kinds_of_complexity()
+simulation = post.StaticMechanicalSimulation(example_path)
+
+# print the simulation to get an overview of what's available
+print(simulation)
+
+# mesh_info = simulation.mesh_info  # TODO: expose MeshSelectionManager?
+# # print(mesh_info)
+
+###############################################################################
+# Get the mesh
+# ------------
+
+# Split the global mesh according to mesh properties
+meshes = simulation.split_mesh_by_properties(
+    properties=[elemental_properties.material, elemental_properties.element_shape]
+)
+meshes.plot()
+
+# Split the global mesh and select meshes for specific property values
+print(meshes)
+meshes = simulation.split_mesh_by_properties(
+    properties={
+        elemental_properties.material: 1,
+        elemental_properties.element_shape: [0, 1],
+    }
+)
+
+# Mesh<index=0, num_nodes=100, num_elem=1000>
```

Comment:
Since this is an example shouldn't we remove unnecessary comments?

Reply:
Could you also make sure that we display properly the results of each API call in the generated doc?